### PR TITLE
Fix warning

### PR DESCRIPTION
**Pull request by jasonfan1997**
Ignore known warning in threeml model and fix the sindec problem

### CODE REVIEW

File: mla/threeml/models.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def _init_sob_ratio(self, *args, **kwargs) -> None:
 
         # Normalize histograms by dec band
         with warnings.catch_warnings():  # divide zero warnings
-            warnings.simplefilter("ignore")  # we don't use those bins
+            warnings.simplefilter('ignore')  # we don't use those bins
```
Comment (thejevans) [inline mla/threeml/models.py:126]: please use np.errstate instead. look at __post_init__() in I3EventModel for an example

Comment (thejevans) [reply mla/threeml/models.py]: I fixed this in a new commit to this branch

File: mla/threeml/models.py [hunk 2]
```diff
@@ -122,30 +121,35 @@ def _init_sob_ratio(self, *args, **kwargs) -> None:
...
-                ratio[i] = spline(bin_centers)
-            elif len(x_good) == 1:
-                ratio[i] = y_good
-            else:
-                ratio[i] = 0
+
+        with np.errstate(divide='ignore'):  # divide zero warnings
+            ratio = sig_h / self._background_sob_map
+
+        with np.errstate(divide='ignore'):  # divide zero warnings
```
Comment (thejevans) [inline mla/threeml/models.py:137]: is this the right filter you want here? please change if necessary (same goes for other instances)

Comment (jasonfan1997) [reply mla/threeml/models.py:137]: Yes. I think you can merge it now.

**Pull request merged**